app.js

```diff
@@ -1,9 +1,23 @@
 const express = require("express");
 const { getTopics } = require("./controllers/topics.controllers");
+const { getApi } = require("./controllers/api.controllers");
+const { getArticleById } = require("./controllers/articles.controllers");
 const app = express();
 
 app.use(express.json());
 
+app.get("/api", getApi);
+
 app.get("/api/topics", getTopics);
 
+app.get("/api/articles/:article_id", getArticleById);
+
+app.use((err, req, res, next) => {
+  if (err.status === 404) {
```

Comment:
if we just check that the status is truthy then this error handler is can be dynamic for all future Promise.rejects which you create in future.